lms/djangoapps/instructor/urls.py

```diff
@@ -0,0 +1,15 @@
+"""
+Instructor API endpoint urls.
+"""
+
+from django.conf.urls import url, include
+
+from openedx.core.constants import COURSE_ID_PATTERN
+from .views import api_urls
+
+
+urlpatterns = [
+    url(rf'^courses/{COURSE_ID_PATTERN}/instructor/api/', include(api_urls.urlpatterns)),
+    url(rf'^instructor/api/v1/courses/{COURSE_ID_PATTERN}/',
```

Comment:
@xitij2000, I guess if we are having any API endpoints in Open edX then its URL naming should be like `/api/...` as per this [edX REST API Conventions](https://openedx.atlassian.net/wiki/spaces/AC/pages/18350757/edX+REST+API+Conventions#1.-URL-Naming).

This practice will make API endpoints available at https://courses.edx.org/api-docs/ (and at Open edX instances, too) so that community can refer to it easily rather than digging the code-base.

Reply:
Thanks I've updated the paths to match the conventions.